### PR TITLE
🧹 Adding logging to configureReceiveConfig

### PR DESCRIPTION
**Pull request by ravinagill15**
## In this PR:
- Added logging to `configureReceiveConfig` in `ua-devtools > src > oapp > config.ts`

### CODE REVIEW

File: packages/ua-devtools/src/oapp/config.ts [hunk 1]
```diff
@@ -361,56 +361,79 @@ export const configureSendConfig: OAppConfigurator = withOAppLogger(
...
-    for (const {
-        vector: { from, to },
-        config,
-    } of graph.connections) {
-        if (config?.receiveConfig?.ulnConfig == null) {
-            continue
-        }
+export const configureReceiveConfig: OAppConfigurator = withOAppLogger(
+    async (graph, createSdk) => {
+        const logger = createOAppLogger()
```
Comment (ravinagill15) [inline packages/ua-devtools/src/oapp/config.ts:366]: change 1

File: packages/ua-devtools/src/oapp/config.ts [hunk 2]
```diff
@@ -361,56 +361,79 @@ export const configureSendConfig: OAppConfigurator = withOAppLogger(
...
-            'receiveLibrary has not been set in your config and no default value exists'
-        )
+        // This function builds a map to find all SetConfigParam[] to execute for a given OApp and ReceiveLibrary
+        const setConfigsByEndpointAndLibrary: OmniPointMap<Map<OmniAddress, SetConfigParam[]>> = new OmniPointMap()
+        for (const {
+            vector: { from, to },
+            config,
+        } of graph.connections) {
+            if (config?.receiveConfig?.ulnConfig == null) {
+                logger.verbose(`ulnConfig not set for ${formatOmniVector({ from, to })}, skipping`)
```
Comment (ravinagill15) [inline packages/ua-devtools/src/oapp/config.ts:375]: change 2

File: packages/ua-devtools/src/oapp/config.ts [hunk 3]
```diff
@@ -361,56 +361,79 @@ export const configureSendConfig: OAppConfigurator = withOAppLogger(
...
+            const hasUlnConfig = await endpointSdk.hasAppUlnConfig(
+                from.address,
+                currentReceiveLibrary,
+                to.eid,
+                config.receiveConfig.ulnConfig
             )
+
+            logger.verbose(
+                `Checked Uln configuration for ${formatOmniVector({ from, to })}: ${printBoolean(hasUlnConfig)}`
+            )
```
Comment (ravinagill15) [inline packages/ua-devtools/src/oapp/config.ts:402]: change 3

File: packages/ua-devtools/src/oapp/config.ts [hunk 4]
```diff
@@ -361,56 +361,79 @@ export const configureSendConfig: OAppConfigurator = withOAppLogger(
...
+
+                const updatedConfigList =
+                    setConfigsByEndpointAndLibrary.getOrElse(from, () => new Map()).get(currentReceiveLibrary) ?? []
+                const updatedConfigListCsv = updatedConfigList
+                    .map(
+                        (setConfigParam) =>
+                            '{configType: ' + setConfigParam.configType + ', config: ' + setConfigParam.config + '}'
+                    )
+                    .join(', ')
+                logger.verbose(`Set Uln configuration ${updatedConfigListCsv} for ${formatOmniVector({ from, to })}`)
```
Comment (ravinagill15) [inline packages/ua-devtools/src/oapp/config.ts:425]: change 4

**Review by ravinagill15 — COMMENTED** [review]
Changes highlighted by github are mostly from indentation -- I have commented the actual changes

**Review by janjakubnanista — APPROVED** [review]

**Pull request merged**